### DISCUSSION TITLE
Error while configuring OPENFAST

### DISCUSSION THREAD

**Discussion by joeb8799** [OpenFAST/openfast#1154] [Q&A]
Hello,
I have an issue while configuring OPENFAST, after executing all the lines of code as cited in documentation, i got the following error while executing "make openfast", my gfortran version is the 6th version and i'm working on a linux virtual machine
[ 60%] Building Fortran object modules/hydrodyn/CMakeFiles/hydrodynlib.dir/src/WAMIT.f90.o
[ 60%] Building Fortran object modules/hydrodyn/CMakeFiles/hydrodynlib.dir/src/WAMIT2.f90.o
[ 60%] Building Fortran object modules/hydrodyn/CMakeFiles/hydrodynlib.dir/src/Waves2.f90.o
[ 60%] Building Fortran object modules/hydrodyn/CMakeFiles/hydrodynlib.dir/src/HydroDyn.f90.o
[ 60%] Linking Fortran static library libhydrodynlib.a
[ 60%] Built target hydrodynlib
Scanning dependencies of target hydrodyn_driver
[ 61%] Building Fortran object modules/hydrodyn/CMakeFiles/hydrodyn_driver.dir/src/HydroDyn_DriverCode.f90.o
[ 61%] Linking Fortran executable hydrodyn_driver
[ 61%] Built target hydrodyn_driver
Scanning dependencies of target orcaflexlib
[ 61%] Building Fortran object modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/src/OrcaFlexInterface_Types.f90.o
[ 62%] Building Fortran object modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/src/OrcaFlexInterface.f90.o
‘
/home/g69581/OpenFAST/modules/orcaflex-interface/src/OrcaFlexInterface.f90:761:0:

       CALL C_F_PROCPOINTER( p%DLL_Orca%ProcAddr(2), OrcaDLL_Calc )
 
in pp_format, at pretty-print.c:633
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-6/README.Bugs> for instructions.
modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/build.make:62 : la recette pour la cible « modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/src/OrcaFlexInterface.f90.o » a échouée
make[2]: *** [modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/src/OrcaFlexInterface.f90.o] Erreur 1
CMakeFiles/Makefile2:1202 : la recette pour la cible « modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/all » a échouée
make[1]: *** [modules/orcaflex-interface/CMakeFiles/orcaflexlib.dir/all] Erreur 2
Makefile:127 : la recette pour la cible « all » a échouée
make: *** [all] Erreur 2

**Comment by rafmudaf** [comment]
This sounds more like a question than a bug so I've converted the issue to a discussion thread. I am not sure exactly what is going on, so I'll ask some questions to get more information from you.

---

> after executing all the lines of code as cited in documentation

What portion of the docs are you looking at?

What flavor and version of Linux are you using?

Are you tied to GCC 6? Newer versions like 11 are much better.

Compiling is only generally required if you are changing the source code or compiling for integration with Simulink. Do you need to compile or can you install via [conda-forge](https://openfast.readthedocs.io/en/main/source/install/index.html#conda-installation).

**Reply by joeb8799** [reply]
Well i'm using a linux 4.9.0-13-amd64 x86 64 and I am tied to GCC 6.
Let me explain to you what i am doing:
I am working on enhancing the parameters of the ROSCO controller in order to get a better control and that is done by applying a method called Robust scheduling that is already present in the ROSCO controller. This method take the linearization files (.lin) as an input. I need to run the openfast executable in order to generate these .lin files so that i can apply robust scheduling to a turbine model i have.
So first of all, i need to install Openfast and then run the executable with the fst file of my turbine model.
I followed the instructions present in the following link https://openfast.readthedocs.io/en/main/source/install/index.html and when i arrived to configuring openfast, i got the issue that i already sent.
